.aws/graasp-dev.json

```diff
@@ -109,9 +109,9 @@
       "entryPoint": null,
       "portMappings": [
         {
-          "hostPort": 8080,
+          "hostPort": 8081,
```

Comment:
Why do we need this change exactly? Can you check with @spaenleh if it's 100% needed.